scopes/react/react/react.env.ts

```diff
@@ -44,7 +46,7 @@ const buildTsConfig = require('./typescript/tsconfig.build.json');
 /**
  * a component environment built for [React](https://reactjs.org) .
  */
-export class ReactEnv implements Environment {
+export class ReactEnv implements TesterEnv, LinterEnv, DevEnv, BuilderEnv, DependenciesEnv, PackageEnv {
```

Comment:
Nice! :)
just making sure, doesn't it also implement `Environment` or it's not needed anymore?

Reply:
they all extend Environment, so it's not needed